AdvanceSwiftExercise.playground/Pages/Average.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,13 @@
+var grade1 = 7.0
+var grade2 = 9.0
+var grade3 = 5.0
+var yourGrade = 8.0
+
+var avg = (grade1  + grade2 + grade3 + yourGrade)/4
+if yourGrade > avg{
```

Comment:
proper spacing after 'avg' and you can do this if else condition in one line also.